packages/common/src/hardhat/tasks/artifacts.ts

```diff
@@ -5,6 +5,12 @@ import { task, types } from "hardhat/config";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 import { CombinedHRE } from "./types";
 
+// Assumes these packages have the following structure:
```

Comment:
this is much better than how you had it before as we can expand to `@across-protocol/contracts-v2` and all future projects that follow this separation pattern.